### PR TITLE
Disable QDS on UI

### PR DESCRIPTION
**Pull request by stuti149**
image

### CODE REVIEW

File: extensions/sql-migration/src/constants/strings.ts [hunk 1]
```diff
@@ -92,7 +92,7 @@ export const DATABASE_FOR_ASSESSMENT_DESCRIPTION = localize('sql.migration.datab
 
 // XEvents assessment
 export const XEVENTS_ASSESSMENT_TITLE = localize('sql.migration.database.assessment.xevents.title', "Assess Ad-hoc or dynamic SQL");
-export const XEVENTS_ASSESSMENT_DESCRIPTION = localize('sql.migration.database.assessment.xevents.description', "For the selected databases, select the option you want to assess ad-hoc or dynamic SQL queries or any DML statements initiated through the application data layer. {0}");
+export const XEVENTS_ASSESSMENT_DESCRIPTION = localize('sql.migration.database.assessment.xevents.description', "For the selected databases, optionally provide extended event session files to assess ad-hoc or dynamic SQL queries or any DML statements initiated through the application data layer.");
```
Comment (Ramudaykumar) [inline extensions/sql-migration/src/constants/strings.ts:95]: This change was needed for showing previous message, right? will need to revert this change as well while enabling it for next release?

**Review by Ramudaykumar — APPROVED** [review]
Looks good me, approving it with question. No change needed in PR.

**Comment by stuti149** [comment]
> Looks good me, approving it with question. No change needed in PR.

yes
